src/main/java/com/arpnetworking/metrics/mad/model/DefaultRecord.java

```diff
@@ -100,13 +107,15 @@ private DefaultRecord(final Builder builder) {
         _metrics = builder._metrics;
         _id = builder._id;
         _time = builder._time;
+        _request_time = Optional.ofNullable(builder._request_time);
         _annotations = builder._annotations;
         _dimensions = builder._dimensions;
     }
 
     private final ImmutableMap<String, ? extends Metric> _metrics;
     private final String _id;
     private final ZonedDateTime _time;
+    private final ZonedDateTime _request_time;
```

Comment:
We typically make the member `Optional` to ensure any internal logic is also forced to deal with its optionality as well as to avoid creating a new `Optional` on every access.